pkgs/by-name/q2/q2pro/package.nix

```diff
@@ -19,33 +19,36 @@
   libdecor,
   ffmpeg,
   wayland-scanner,
-  makeWrapper,
+  makeBinaryWrapper,
   versionCheckHook,
+  copyDesktopItems,
+  makeDesktopItem,
+  desktopToDarwinBundle,
   x11Support ? stdenv.hostPlatform.isLinux,
   waylandSupport ? stdenv.hostPlatform.isLinux,
 }:
 
 stdenv.mkDerivation (finalAttrs: rec {
   pname = "q2pro";
-  version = "3510";
+  version = "3624"; # git rev-list --count HEAD
 
   src = fetchFromGitHub {
     owner = "skullernet";
     repo = "q2pro";
-    rev = "refs/tags/r${version}";
```

Comment:
The r{xxxx} tags are not more significant than any other commits in master and shouldn't be considered as a form of releases - https://github.com/skullernet/q2pro/issues/363#issuecomment-2442448224